src/configs/rollup.es5.config.js

```diff
@@ -39,8 +39,15 @@ const minify = require('rollup-plugin-terser').terser
 const license = require('rollup-plugin-license')
 const os = require('os')
 const extensions = ['.js', '.ts', '.mjs']
+const deepMerge = require('deepmerge')
 
-module.exports = {
+let customConfig
+
+if (process.env.LNG_CUSTOM_ROLLUP === 'true') {
+  customConfig = require(path.join(process.cwd(), 'rollup.es5.config'))
```

Comment:
What about the _env_?

Reply:
Hey @ricardo-a-alves-alb ,
The ones that are mentioned in the env will be ignored here and the custom config takes the priority here.